RCAIDE/Visualization/Geometry/plot_propeller.py

```diff
@@ -0,0 +1,84 @@
+## @ingroup Plots-Geometry
```

Comment:
This script is pretty much a copy and paste. We'll deal with plots in another PR. Remove it for now, but hold on to a copy.

Reply:
This one got lost, meant to remove it. Thanks.